### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tinect**
Version 6.1 shouldn't be advertised :-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,18 +15,19 @@ you should use the [development template](https://github.com/shopware/developmen
 In each commit a composer.lock is contained to ensure that the version being
 deployed is the version that was tested in our CI. We currently provide two
 branches:
-- `6.1`: stable patch releases (`v6.1.0-rc2`, `v6.1.0`, `v6.1.19`, `v6.1.*`, but not `v6.2.0`)
-- `master`: stable minor+patch releases (`v6.1.3`, `v6.1.15`, `v6.2.0`, `v6.3.0`...)
+- `6.1`: stable patch releases (`v6.1.*`)
```
Comment (mitelg) [inline README.md:18]: could we make the following line more general? otherwise we would have to change it every major release

File: README.md [hunk 2]
```diff
@@ -15,18 +15,19 @@ you should use the [development template](https://github.com/shopware/developmen
...
+- `6.2`: stable patch releases (`v6.2.*`)
+- `master`: stable minor+patch releases (`v6.1.3`, `v6.1.15`, `v6.2.0`, `v6.3.0`, ...)
 
-The `6.1` branch contains all the 6.1 releases. It's stable now and only gets non-breaking bug fixes. (security issues are an exception).
+Stable patch releases are stable and only get non-breaking bug fixes. (security issues are an exception).
```
Comment (mitelg) [inline README.md:22]: stable release are stable? :wink:

Comment (tinect) [reply README.md:22]: All the stable stuff is stable, for sure... You're right xD

File: README.md (hunk 1 above)
Comment (tinect) [reply README.md:18]: Any idea? :-)

Comment (tinect) [reply README.md:18]: An "x" for minor ? 🤔

**Comment by shyim** [comment]
Has been already fixed. Anyway thanks!

**Pull request closed without merging**